src/Common/src/Interop/Unix/libc/Interop.getegid.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class libc
+    {
+        [DllImport(Libraries.Libc)]
+        internal static extern int getegid();
+    }
+}
```

Comment:
Perhaps just a github thing but there seems to be an inconsistency with the newlines at the end of the files.